cmd/config.go

```diff
@@ -245,8 +245,15 @@ func (d *ConfigDuration) UnmarshalYAML(unmarshal func(interface{}) error) error
 
 // GRPCClientConfig contains the information needed to talk to the gRPC service
 type GRPCClientConfig struct {
+	// ServerAddress is a single host:port combination that the gRPC client
+	// will, if necessary, resolve via DNS and then connect to. This field
+	// cannot be used in combination with `ServerIPAddresses` field.
 	ServerAddress string
-	Timeout       ConfigDuration
+	// ServerIPAddresses is a list of IPv4/6 addresses, in the format IPv4:port,
+	// IPv6:port or :port, that the gRPC client will connect to. This field
+	// cannot be used in combination with `ServerAddress`.
```

Comment:
This should document that the server's certificate will be validated against this IP address, so it must have appropriate IP addresses in it.